index.js

```diff
@@ -75,7 +76,18 @@ var swaggerInitFunction = function (swaggerDoc, opts) {
       next()
     }
   }
-} 
+}
+
+var swaggerAssetMiddleware = options => {
+  var staticServer = express.static(swaggerUi.getAbsoluteFSPath(), options || {})
+  return (req, res, next) => {
+    if(/\.(js|css|map|png|html)$/.test(req.path)) {
```

Comment:
This prevents the app from serving `swagger-ui-dist`'s `index.html` instead of our own.